### PR TITLE
Add image loaders to clevr dataset

### PR DESCRIPTION
**Pull request by efahnestock**

### CODE REVIEW

File: experimental/overhead_matching/learned/data/clevr_dataset.py [hunk 1]
```diff
@@ -1,27 +1,85 @@
...
 import json
 from collections import defaultdict
 
-from typing import Callable
+from typing import Callable, NamedTuple
 
+DEFAULT_CLEVR_OVERHEAD_TRANSFORM = torch.nn.Sequential(
+    tf.ToImage(),
+    tf.ToDtype(torch.float32, scale=True),
+    tf.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
```
Comment (ewfuentes) [inline experimental/overhead_matching/learned/data/clevr_dataset.py:15]: pull normalization out?

File: experimental/overhead_matching/learned/data/clevr_dataset.py [hunk 2]
```diff
@@ -1,27 +1,85 @@
...
+)
+DEFAULT_CLEVR_EGO_TRANSFORM = torch.nn.Sequential(
+    tf.ToImage(),
+    tf.ToDtype(torch.float32, scale=True),
+    tf.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
```
Comment (ewfuentes) [inline experimental/overhead_matching/learned/data/clevr_dataset.py:20]: pull normalization out?

File: experimental/overhead_matching/learned/data/clevr_dataset.py [hunk 3]
```diff
@@ -1,27 +1,85 @@
...
 
-def collator(objs):
+
+class CleverDatasetItem(NamedTuple):
+    scene_description: dict | None = None
+    overhead_image: torch.Tensor | None = None
+    ego_image: torch.Tensor | None = None
+
+    def __repr__(self):
+        return f"ClevrDatasetItem(scene_description: {self.scene_description}, overhead_image: {self.overhead_image}, ego_image: {self.ego_image})"
```
Comment (ewfuentes) [inline experimental/overhead_matching/learned/data/clevr_dataset.py:30]: Maybe print number of objects (or object types) and image dimensions?

Comment (ewfuentes) [reply experimental/overhead_matching/learned/data/clevr_dataset.py:30]: could be in a follow up.

**Review by ewfuentes — APPROVED** [review]
